### DISCUSSION TITLE
Order Now : Cafe Search

### DISCUSSION THREAD

**Discussion by c4coderandcreator** [c4coderandcreator/Flavour-Fusion#125] [Ideas]
Hey @VIDYA1980  can we separate by date result on left and By cafe result on right also user should click the cafe to see if they have ordered from there or not !! Feel free to share your ideas over this !! Waiting for your response !!

**Comment by VIDYA1980** [comment]
Hey,I think maybe this can be added as a feature ,and the previous pr will be accepted for hacktoberfest ryt?

**Comment by c4coderandcreator** [comment]
> Hey,I think maybe this can be added as a feature ,and the previous pr will be accepted for hacktoberfest ryt?

Dont worry about the label @VIDYA1980  !! You please focus on contribution !! Yes it is eligible for hacktoberfest !!

**Reply by VIDYA1980** [reply]
i will work over it and create a new pr

**Comment by c4coderandcreator** [comment]
@VIDYA1980 check this in issue , i have created this for you !! ask for assignment there !!
